inspector.el

```diff
@@ -806,10 +806,17 @@ When PRESERVE-HISTORY is T, inspector history is not cleared."
     (with-current-buffer-window "*inspector pprint*"
         nil nil
       (local-set-key "q" #'kill-this-buffer)
+      ;; For evil mode use `q to kill the buffer
+        (when (fboundp 'evil-local-set-key)
+          (evil-local-set-key 'normal "q" #'kill-this-buffer))
+
       (let ((pp-use-max-width inspector-pp-use-max-width)
             (pp-max-width inspector-pp-max-width))
         (ignore pp-use-max-width pp-max-width)
-        (pp object)))))
+        (pp object)
+        ;; Jump to this buffer
+       (switch-to-buffer-other-window "*inspector pprint*")
```

Comment:
This is optional, I just realize that when use `pprint` function it is not auto jump to result window

Reply:
I think it is OK to jump.